protocols/discord.py

```diff
@@ -482,7 +495,14 @@ def _update_user_status(self, guild_id, uid, presence):
 
     @Plugin.listen('PresenceUpdate')
     def on_presence_update(self, event, *args, **kwargs):
-        self._update_user_status(event.guild_id, event.presence.user.id, event.presence)
+        self._update_user_status(event.guild, event.presence.user.id, event.presence)
+
+    @staticmethod
+    def _should_burst_user(pylink_netobj, user):
+        return not pylink_netobj.join_offline_users and \
+               user.presence != UNSET and \
+               user.presence.status not in [DiscordStatus.OFFLINE, DiscordStatus.INVISIBLE]
```

Comment:
Could we rewrite this as an `or`? I think it's easier to read

```python
    # Burst users only if join_offline_users is enabled or their status is online/away/DND
    return pylink_netobj.join_offline_users or (user.presence != UNSET and \
        user.presence.status not in [DiscordStatus.OFFLINE, DiscordStatus.INVISIBLE])
```

Reply:
I personally prefer if statements being only one level deep, but I can swing either way, here

Reply:
I think logically clearer is better